### ISSUE TITLE
PhylotaR Cluster error: blastn failed to run

### ISSUE THREAD

**Issue by maariapaul** [ropensci/phylotaR#44]
Hi! I've been trying to use phylotaR with the code:

```
wd <- 'C:/Users/maari/Desktop/Trial'
ncbi_dr <- 'C:/Program Files/NCBI/blast-2.9.0+/bin'
txid <- 231623
setup(wd = wd, txid = txid, ncbi_dr = ncbi_dr, v = T)
run(wd = wd)
```
Everything seems to work fine until the Cluster stage, where I get the following error:

Error in stages_run(wd = wd, frm = 1, to = nstages, stgs_msg = stgs_msg) : 
  Unexpected Error : blastn failed to run. Check BLAST log files.

BLAST query/options error: '"6' is not a valid output format
Please refer to the BLAST+ user manual.

Could you help solve this error?
I've already tried running the code with different taxa, R and Blast versions, but always seem to get an error in the cluster stage.
I've also tried installing the most recent version of phylotaR using the command:
`install.packages('phylotaR')`
I'm working on a windows 10 machine.

**Comment by DomBennett** [comment]
Hi!

This looks similar to an error I fixed. Try installing the latest version from GitHub with the library `remotes`:

```r
library(remotes)
install_github('ropensci/phylotar')
```

**Comment by maariapaul** [comment]
I've tried installing phylotaR with the code suggested, but I get the following error:

Error: Failed to install 'phylotaR' from GitHub:
  (converted from warning) package ‘restez’ is not available (for R version 3.4.4)

Has the package been removed from the CRAN repository or do I simply need to use another R version?

**Comment by DomBennett** [comment]
Ah whoops.... `restez` got removed from CRAN due to a key dep being archived. Try:

```r
library(remotes)
# install restez dep
install_github("hannesmuehleisen/MonetDBLite-R")
# install restez
install_github("ropensci/restez")
# install phylotaR
install_github('ropensci/phylotar')
```

**Comment by maariapaul** [comment]
I cannot seem to install MonetDBLite. I get the following error:

Error: Failed to install 'MonetDBLite' from GitHub:
  (converted from warning) running command '"C:/PROGRA~1/R/R-34~1.1/bin/x64/R" CMD INSTALL -l "C:\Users\maari\OneDrive\Documentos\R\win-library\3.4" "C:/Users/maari/AppData/Local/Temp/Rtmpqm9Xz8/file20b45323f55/MonetDBLite_0.6.1.tar.gz"' had status 1
In addition: Warning message:
In missing_devel_warning(pkgdir) :
  Package MonetDBLite has compiled code, but no suitable compiler(s) were found. Installation will likely fail.
  Install Rtools (https://cran.r-project.org/bin/windows/Rtools/).Then use the pkgbuild package, or make sure that Rtools in the PATH.

Any idea how I can solve this?

**Comment by DomBennett** [comment]
Hi!

The error message is asking you to install "Rtools". This is an additional software, specific to Windows, that some R packages require. (It is also required for buidling and developing packages.)

Go to this link (https://cran.r-project.org/bin/windows/Rtools/), install Rtools and then try to reinstall MonetDBLite.

Thanks!

**Comment by maariapaul** [comment]
Hi,

I've managed to make it work, turns out the problem was in the path to Rtools.

But now, the setup doesn't run smoothly. I've used the same code:

```
library(phylotaR)
wd <- 'C:/Users/maari/Desktop/1. LIFELINE/phylotaR/Rotaria'
ncbi_dr <- 'C:/Program Files/NCBI/blast-2.9.0+/bin'
txid <- 231623
setup(wd = wd, txid = txid, ncbi_dr = ncbi_dr, v = T)
```

And I get the error:

-------------------------------------------------
phylotaR: Implementation of PhyLoTa in R [v1.2.0]
-------------------------------------------------
Checking for valid NCBI BLAST+ Tools ...
Found: [C:/Program Files/NCBI/blast-2.9.0+/bin/makeblastdb]
Found: [C:/Program Files/NCBI/blast-2.9.0+/bin/blastn]
. . Running makeblastdb
Error: makeblastdb failed to run. Check BLAST log files.

I've checked and the blast folder has no log, only a file called testdbfl.
